pkg/client/client.go

```diff
@@ -62,7 +62,17 @@ func NewClient(token, apiEndpoint string) (runtimeclient.Client, error) {
 }
 
 func NewClientFromRestConfig(config *rest.Config) (runtimeclient.Client, error) {
-	config.Insecure = true
+	tc, err := config.TransportConfig()
```

Comment:
Do we have to do this? Shouldn't we just rely on what's in the kubeconfig to init the connection?

Reply:
yup it's gone! thanks!